kiva/tests/test_ps_drawing.py

```diff
@@ -23,5 +23,6 @@ def draw_and_check(self):
         if not any((line.endswith("fill"),
                     line.endswith("stroke"),
                     line.endswith("cliprestore"),
+                    line.endswith("grestore"),
```

Comment:
not sure what this special incantation is either.

Reply:
This is what shows up in the postscript output when you draw an image.